prometheus/exporter/job_exporter.py

```diff
@@ -82,8 +78,11 @@ def gen_job_metrics(logDir, gpuMetrics):
         containerCPUPerc = 'container_CPUPerc{{{0}}} {1}\n'.format(labelStr, stats[container]["CPUPerc"])
         containerMemUsage = 'container_MemUsage{{{0}}} {1}\n'.format(labelStr, stats[container]["MemUsage_Limit"]["usage"])
         containerMemLimit = 'container_MemLimit{{{0}}} {1}\n'.format(labelStr, stats[container]["MemUsage_Limit"]["limit"])
-        containerNetIn = 'container_NetIn{{{0}}} {1}\n'.format(labelStr, stats[container]["NetIO"]["in"])
-        containerNetOut = 'container_NetOut{{{0}}} {1}\n'.format(labelStr, stats[container]["NetIO"]["out"])
+        inSize, outSize = network.acc_per_container_network_metrics(connectionDic, pid)
+        inBandwidth = inSize / 40
+        outBandwidth = outSize / 40
```

Comment:
Why divided by 40 ?

Reply:
use the each last 40s accumulate bytes from iftop cmd / 40 will get bytes per second